paf-mvp-pattern-library-audit/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "OneKey",
+  "version": "1.0.0",
+  "description": "",
+  "main": "build/ok-ui.js",
+  "scripts": {
+    "start": "npm run storybook",
+    "build": "npm run build-css && mkdir -p build/images && cp assets/images/OneKey.svg build/images/OneKey.svg && cp assets/images/OneKeyIcon.svg build/images/OneKeyIcon.svg",
+    "build-test": "open build/index.html",
+    "build-js": "webpack --mode production --entry ./javascript/ok-ui.js --output-filename ok-ui.js -o ./build",
+    "build-css": "sass styles/ok-ui-audit.scss:build/ok-ui-audit.css --style compressed",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@babel/core": "^7.17.0",
+    "@babel/plugin-proposal-class-properties": "^7.16.7",
+    "@babel/plugin-proposal-decorators": "^7.17.0",
+    "@babel/plugin-proposal-export-default-from": "^7.16.7",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.16.7",
+    "@babel/plugin-proposal-object-rest-spread": "^7.16.7",
+    "@babel/plugin-proposal-optional-chaining": "^7.16.7",
+    "@babel/plugin-proposal-private-methods": "^7.16.11",
+    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
+    "@babel/plugin-transform-arrow-functions": "^7.16.7",
+    "@babel/plugin-transform-block-scoping": "^7.16.7",
+    "@babel/plugin-transform-classes": "^7.16.7",
+    "@babel/plugin-transform-destructuring": "^7.16.7",
+    "@babel/plugin-transform-for-of": "^7.16.7",
+    "@babel/plugin-transform-parameters": "^7.16.7",
+    "@babel/plugin-transform-shorthand-properties": "^7.16.7",
+    "@babel/plugin-transform-spread": "^7.16.7",
+    "@babel/preset-env": "^7.16.11",
+    "@babel/preset-typescript": "^7.16.7",
+    "@storybook/addon-actions": "^6.4.18",
+    "@storybook/addon-essentials": "^6.4.18",
+    "@storybook/addon-links": "^6.4.18",
+    "@storybook/html": "^6.4.18",
+    "babel-loader": "^8.2.3",
+    "babel-plugin-macros": "^3.1.0",
+    "babel-plugin-polyfill-corejs3": "^0.5.2",
+    "core-js": "^3.21.0",
+    "css-loader": "^5.2.7",
+    "sass": "^1.49.9",
+    "sass-loader": "^10.1.1",
```

Comment:
I had to install `@mdx-js/react` to be able to start Storybook.
Isn't missing from the dependencies?

Reply:
Also, NIT but the `package-lock.json` file seems to have been generated with a version of node that is too old: it should be in version 2 for node 17 (which is the version used on the repo, see `.nvmrc`)

Reply:
Re: Package - We've not had to install additional packages. I guess there is no harm in adding it but I can't recreate the issue.
Re: Version - We tend not to use the odd versions of Node because they aren’t the stable releases. We’ve opted for v16 as although v18 was recently released it doesn’t appear to be immediately compatible with our version of Storybook.